scripts/src/sanitycheckpr/sanitycheckpr.py

```diff
@@ -23,8 +25,8 @@ def ensure_only_chart_is_modified(api_url, repository, branch):
     files_api_url = f'{api_url}/files'
     headers = {'Accept': 'application/vnd.github.v3+json'}
     r = requests.get(files_api_url, headers=headers)
-    pattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/(.+?(?=\/))/.*")
-    reportpattern = re.compile(r"charts/(\w+)/([\w-]+)/([\w-]+)/([\w\.]+)/report.yaml")
+    pattern = re.compile(r"charts/"+TYPE_MATCH_EXPRESSION+"/([\w-]+)/([\w-]+)/"+SEMVER_MATCH_EXPRESSION+"/.*")
```

Comment:
In the original pattern it will look into any nested directories of any depth >= `charts/(\w+)/([\w-]+)/([\w-]+)/`, after the change it will only look into one more depth under `charts/(\w+)/([\w-]+)/([\w-]+)/` right? So anything with more depth will not match and fail the sanity check.

Reply:
The /.* at the end takes care of that. Also see my tests

Reply:
see: ('PASS - matched :', 'charts/partners/test-org1/ibm-object-storage-plugin/2.1.2/src/templates/tests/service.yaml')

Reply:
that cleared my question, thank you. good to merge for me

Reply:
After version we want the repor right, why would we want more depth after that?